src/main/java/com/rettichlp/unicacityaddon/events/faction/polizei/WantedListEventHandler.java

```diff
@@ -0,0 +1,164 @@
+package com.rettichlp.unicacityaddon.events.faction.polizei;
+
+import com.rettichlp.unicacityaddon.base.config.ConfigElements;
+import com.rettichlp.unicacityaddon.base.registry.annotation.UCEvent;
+import com.rettichlp.unicacityaddon.base.text.ColorCode;
+import com.rettichlp.unicacityaddon.base.text.Message;
+import com.rettichlp.unicacityaddon.base.text.PatternHandler;
+import net.minecraftforge.client.event.ClientChatReceivedEvent;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+
+import java.util.regex.Matcher;
+
+/**
+ * @author Gelegenheitscode
+ */
+
+@UCEvent
+public class WantedListEventHandler {
+    @SubscribeEvent
+    public void onChat(ClientChatReceivedEvent e)   {
+        if (!ConfigElements.getBetterWantedList())  {
+            return;
+        }
+        String msg = e.getMessage().getUnformattedText();
+
+        Matcher m = PatternHandler.WANTED_LIST_PATTERN.matcher(msg);
+        if (m.find())   {
```

Comment:
Geht gar nicht... zu hohe Komplexität und völlig unnötige Abfragen... Die Nachricht zu verändern würde auch das NameTag Providing zerstören, da reden wir nochmal drüber. Da du dummerweise aber alle 3 Feature in eine PR gepackt hast, ist dadurch halt alles abgelehnt...

Am besten ist es wenn du für jedes feature eine PR machst